src/main/java/com/spotify/confidence/ConfidenceValue.java

```diff
@@ -120,8 +121,28 @@ public static ConfidenceValue of(boolean value) {
     return new BooleanValue(value);
   }
 
-  public static List of(java.util.List<ConfidenceValue> values) {
-    return new List(values);
+  public static ConfidenceValue.List ofStrings(java.util.List<String> values) {
+    return new List(values.stream().map(ConfidenceValue::of).collect(Collectors.toList()));
+  }
+
+  public static ConfidenceValue.List ofBooleans(java.util.List<Boolean> values) {
+    return new List(values.stream().map(ConfidenceValue::of).collect(Collectors.toList()));
+  }
+
+  public static ConfidenceValue.List ofIntegers(java.util.List<java.lang.Integer> values) {
+    return new List(values.stream().map(ConfidenceValue::of).collect(Collectors.toList()));
+  }
+
+  public static ConfidenceValue.List ofDoubles(java.util.List<java.lang.Double> values) {
+    return new List(values.stream().map(ConfidenceValue::of).collect(Collectors.toList()));
+  }
+
+  public static ConfidenceValue.List ofTimestamps(java.util.List<Instant> values) {
+    return new List(values.stream().map(ConfidenceValue::of).collect(Collectors.toList()));
+  }
+
+  public static ConfidenceValue.List ofDates(java.util.List<LocalDate> values) {
+    return new List(values.stream().map(ConfidenceValue::of).collect(Collectors.toList()));
```

Comment:
Not sure that I am married to what I'm gonna propose but I'm gonna drop the thought anyway; what do you think about having these `ofXs`-functions on the `List` class instead so they will be created with:
```
ConfidenceValue.List.ofStrings(...)
ConfidenceValue.List.ofDates(...)
etc
```

Reply:
I added a new commit that changes the accepted types in `ofStrings` (etc.). The idea is for the user to build the list with ConfidenceValues rather than Java primitives. Although a bit more verbose, I think this is a much nicer API that maps well with the event schema in the backend. I am not sure if this impacts your suggestion on the method's naming though

Reply:
However, the new commit generates quite a lot of warnings at the moment. Maybe there is still room for improvements

Reply:
It was very much a nit and after testing it out locally I think the API you provide here makes sense. Resolving this.